### PR TITLE
Added example on how to actively send messages

### PR DESCRIPTION
**Pull request by derhackler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,24 @@ pid = Hedwig.whereis("jeeves")
...
+## Sending Messages
+
+```elixir
+# Get the pid of the robot
+pid = Hedwig.whereis("alfred")
+
+# Create a Hedwig message
+msg = %Hedwig.Message{
+      type: "groupchat",
+      room: room_jid, # eg. 123456_someroom@conf.hipchat.com"
```
Comment (scrogson) [inline README.md]: Let's not make this HipChat or XMPP specific if possible.

**Comment by scrogson** [comment]
@derhackler thank you very much for this additional. I left a small comment.

**Comment by scrogson** [comment]
:+1:

**Comment by derhackler** [comment]
Yeah. I think this is my 2nd pull request ever :-)

**Pull request merged**